app/views/user_mailer/activation_email.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Welcome to YouTube Emporium!</h1>
+
+<h3>
+ <%= link_to 'Visit here to activate your account.', "http://localhost:3000/activation/#{@user.id}" %>
+</h3>
```

Comment:
does this link need to be changed to a particular route? localhost will not be the url in production